src/test/java/edu/hm/hafner/util/StringContainsUtilsTest.java

```diff
@@ -25,10 +25,30 @@ void shouldHandleNull() {
     @Test
     void shouldSearchStrings() {
         assertThat(containsAnyIgnoreCase("This is a string text.", "something")).isFalse();
-
         assertThat(containsAnyIgnoreCase("This is a string text.", "This")).isTrue();
         assertThat(containsAnyIgnoreCase("This is a string text.", "this")).isTrue();
         assertThat(containsAnyIgnoreCase("This is a string text.", "wrong", "is")).isTrue();
         assertThat(containsAnyIgnoreCase("This is a string text.", "wrong", "IS")).isTrue();
     }
+
+    @Test
+    void shouldHandleNull2() {
```

Comment:
Hier wäre als Methodenname containsAllIgnoreCaseShouldHandleNull besser. Zahlen in Methodennamen, auch bei Tests, sollten vermieden werden.

Reply:
Habe ich im aktuellen Pull Request überarbeitet 😃.
https://github.com/uhafner/codingstyle/pull/140